src/stylesheets/components/_form-input-group.scss

```diff
@@ -1,57 +1,68 @@
 .form-input-group {
   display: flex;
   flex-direction: column;
+  width: rem(320px);
   height: rem(90px);
   margin-bottom: rem(20px);
-  width: rem(320px);
 
   &__label {
-    color: $secondary-color;
-    font-size: map-get($font-sizes, 'md');
+    margin: rem(0 0 10px);
+
+    color: map-get($brand-white, 'primary');
     font-weight: map-get($font-weight, 'bolder');
-    letter-spacing: -0.24px;
+    font-size: map-get($font-sizes, 'md');
     line-height: rem(20px);
-    margin: rem(0 0 10px);
+    letter-spacing: -0.24px;
   }
 
   &__input {
+    display: flex;
     align-items: center;
+    width: 100%;
+    height: rem(55px);
+    margin: auto;
+    padding: 0.375rem 0.75rem;
+
+    color: map-get($brand-white, 'primary');
+    font-weight: map-get($font-weight, 'normal');
+    font-size: map-get($font-sizes, 'lg');
+    line-height: rem(22px);
+    letter-spacing: -0.41px;
```

Comment:
@rafayet-monon I saw that you set `letter-spacing` in many places.
Should we deal with something here? 🤔
In my ideas:
- remove that because it's just a small number of pixels.
- move that as the base font.

Reply:
Removed the negative spacings, Fixed in [6ada2d2](https://github.com/rafayet-monon/surveyor/pull/13/commits/6ada2d22d37cb0fa39af1ba8e59a8880825ee44e)